docs/contributor_guide.md

```diff
@@ -0,0 +1,62 @@
+# Contributor guide
+
+Welcome to the PROJECTNAME Contributor guide!
+
+> If your project is not going to have a docsite (in case it is very simple and having a docsite would be an overkill), the following information should be present in your project CONTRIBUTING.md file and referred to from the [README template](https://github.com/ansible-community/project-template/blob/main/README.md).
+
+> In the following sections, if the content is as simple as a few lines, put it in there, otherwise create separate files and refer to them from the sections. Add other sections if needed.
+
+## Communication
+
+Need help or want to share feedback? See the [Community guide](community_guide.md)!
+
+## Project place in the Ansible ecosystem
+
+> Describe a project place in the Ansible ecosystem here. For example, the project can be a component of another project or serve as an auxiliary project.
+In case the project is a standalone self-sufficient tool, this section can be omitted.
+
+## Non-code contributions
+
+### Ask and answer questions
+
+Ask and answer questions. Reach out and communicate with your fellow users and contributors.
+See the official communication channels in the [Community guide](community_guide.md).
+
+### Contributing to documentation
+
+Reviewing, fixing, and maintaining the documentation is a great way to contribute to PROJECTNAME. 
+
+> Describe how to contribute to the documentation or list references to separate documentation guidelines. Put a GitHub link to issues with documentation-related labels.
+
+### How to open an issue
+
+If you want to report a bug or request a new feature:
+
+1. Search in the [issues](https://github.com/ORG/REPO/issues) for similar reports/requests.
+2. If there are no such issues yet, open a new one by clicking the `New issue` button.
+
+## Code contributions
+
+> Consider having a separate code contributions guide and a link to it from here.
+
+### Project architecture
+
+> Describe your project architecture here: its components and how they work together.
+
+### Getting started
+
+> Create the `easyfix` label in the repository and mark issues that are easy to fix for newcomers.
```

Comment:
The github default is 'good first issue' so we should recommend that here instead of easyfix. The filters we share to find easy issues use both labels.

Reply:
fixed